### ISSUE TITLE
VZE: Possible Bug: Cannot log in after changing password

### ISSUE THREAD

**Issue by sergiogcx** [cityofaustin/atd-vz-data#478]
Daniel Yang reported yesterday that he could not log in the Vision Zero after resetting the password.

This is the screenshot Daniel sees after trying to log once the password is changed:

![thumbnail_image001.png](https://images.zenhubusercontent.com/5b7edad7290aac725aec290c/6816aa05-6a90-46e4-85b8-69fd3bf1ed77)

**Comment by sergiogcx** [comment]
I've tried to recreate the issue by deleting my own account and resetting the password, I've tried with complex passwords that include alphanumeric characters and symbols, and I was able to reset the password without any problems and log in back to VZE also without any problems.

My thoughts to troubleshoot next:
- It may be time to reset the password using another test account.
- One thing I've noticed is the message in yellow presented in the picture that reads "Password login via OIDC-conformant clients with externally-hosted login pages is unsupported. Alternatively, login could have been initiated from the wrong place (e.g., a bookmark)." It could be possible that saving the bookmark to the Auth0 login dialog may contain old access/validation tokens. In other words, we need to make sure we always click the Login button from `https://visionzero.austin.gov/editor/#/login`
